frontend/src/components/regions/UserDashboard/UserDetailedStats.js

```diff
@@ -7,10 +7,15 @@ import {
   TableRow,
   Typography,
 } from "@material-ui/core";
-
+import { colors } from "@material-ui/core";
 import { PieChart, Pie, Cell, Tooltip } from "recharts";
 
-const COLORS = ["#0088FE", "#00C49F", "#FFBB28", "#FF8042"];
+const COLORS = [
+  colors.yellow[600],
```

Comment:
Where did you get these colours from? Should you not be importing them?  I'm sure we use the colours in multiple places?